src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/email/AppellantDecideAnApplicationPersonalisationEmail.java

```diff
@@ -49,18 +51,18 @@ public AppellantDecideAnApplicationPersonalisationEmail(
     @Override
     public String getTemplateId(AsylumCase asylumCase) {
```

Comment:
isPresent on Optionals is procedural style, most often this is replaceable with a functional alternative, e.g. this method could be written as:
 public String getTemplateId(AsylumCase asylumCase) {
        return makeAnApplicationService.getMakeAnApplication(asylumCase, true)
            .map(makeAnApplication -> {
                String decision = makeAnApplication.getDecision();
                boolean isApplicationListed = makeAnApplicationService.isApplicationListed(State.get(makeAnApplication.getState()));
                if (isApplicationListed) {
                    return DECISION_GRANTED.equals(decision) ? decideAnApplicationAppellantGrantedAfterListingEmailTemplateId : decideAnApplicationRefusedAfterListingAppellantEmailTemplateId;
                }
                return DECISION_GRANTED.equals(decision) ? decideAnApplicationAppellantGrantedBeforeListingEmailTemplateId : decideAnApplicationRefusedBeforeListingAppellantEmailTemplateId;
            }).orElse("");
}